cura/Settings/MachineManager.py

```diff
@@ -847,6 +847,24 @@ def variantBuildplateUsable(self) -> bool:
 
         return result
 
+    @pyqtProperty(bool, notify = activeMaterialChanged)
+    def variantCoreUsableForFactor4(self) -> bool:
+        """The selected core is usable if it is in second extruder of Factor4
+        """
+        result = True
+        if not self._global_container_stack:
+            return result
+        if self.activeMachine.id != "UltiMaker Factor 4":
```

Comment:
we can't use this check, creating two factor4's results in the following machine id: `UltiMaker Factor 4 #2`

Reply:
Oh no.

Reply:
fixed here 6542d9ff1334d176d75ed9b2a8e7aba633f88040